package.json

```diff
@@ -28,7 +28,7 @@
     "test:watch": "wtr 'test/**/*.test.*' --node-resolve --watch",
     "lint": "eslint --rulesdir eslint-rules rollup/*.js cem src tasks test wds",
     "lint:fix": "eslint --rulesdir eslint-rules --fix rollup/*.js cem src tasks test wds",
-    "prepack": "npm run test && npm run lint && npm run components:check-lit && npm run components:check-i18n && npm run components:build && npm run components:docs",
+    "prepack": "npm run test && npm run lint && npm run components:check-lit && npm run components:check-i18n",
```

Comment:
this one feels weird but I think you said you needed to do something about this yesterday. Just leaving a comment so you don't forget.

Reply:
I should not have committed this. Thanks.